src/types/mlflow.jl

```diff
@@ -31,5 +31,5 @@ struct MLFlow
     headers::Dict
 end
 MLFlow(baseuri; apiversion=2.0,headers=Dict()) = MLFlow(baseuri, apiversion,headers)
-MLFlow() = MLFlow("http://localhost:5000", 2.0, Dict())
+MLFlow() = MLFlow(ENV["MLFLOW_URI"], 2.0, Dict())
```

Comment:
I don't consider this as a way to go. The default location must be the `mlflow` default location (`http://localhost:5000`).

`MLFlowClient.jl` is just a communication layer, directly related to `mlflow` and not to `MLJ`. Everything related to `MLJ` must be applied in `MLJFlow.jl`. However, the way of adding `ENV["MLFLOW_URI"]` in terms of testing is correct, and discused and implemented in https://github.com/JuliaAI/MLJFlow.jl/pull/30.

Reply:
Done: [Compare 5fac59d to 56479a5](https://github.com/JuliaAI/MLFlowClient.jl/compare/5fac59deeba8a55ff17efa7436bcbff39dfc1b1f..56479a50382a3188f6e98a733285c3e8e04c3875)